Source/FieldSolver/FiniteDifferenceSolver/EvolveE.cpp

```diff
@@ -49,11 +49,21 @@ void FiniteDifferenceSolver::EvolveE (
     std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& Bfield,
     std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& Jfield,
     std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& edge_lengths,
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 > const& face_areas,
+    std::array< std::unique_ptr<amrex::MultiFab>, 3 >& Rhofield,
     std::unique_ptr<amrex::MultiFab> const& Ffield,
     int lev, amrex::Real const dt ) {
 
-   // Select algorithm (The choice of algorithm is a runtime option,
-   // but we compile code for each algorithm, using templates)
+#ifdef AMREX_USE_EB
+    if (m_fdtd_algo != MaxwellSolverAlgo::ECT) {
+        ignore_unused(face_areas, Rhofield);
```

Comment:
We try to write the `amrex::` namespace prefix explicitly as much as possible in new code:
```suggestion
        amrex::ignore_unused(face_areas, Rhofield);
```